app/scripts/modules/core/diffs/commitHistory.component.ts

```diff
@@ -0,0 +1,34 @@
+import {module} from 'angular';
+
+import {IComponentOptions} from 'angular';
+
+export interface ICommit {
+  authorDisplayName: string;
+  commitUrl: string;
+  displayId: string;
+  id: string;
+  message: string;
+  timestamp: number;
+}
+
+class CommitHistoryComponent implements IComponentOptions {
+  public bindings: any = {
+    commits: '<'
+  };
+  public template = `
+  <div class="step-section-details" ng-if="$ctrl.commits && $ctrl.commits.length">
```

Comment:
It was giving padding and margin, but I think is an artifact of playing around with a bunch of different layouts/L&Fs.  I'll remove and see if goes all `GridBagLayout` on me.